docs/async.md

```diff
@@ -64,3 +64,44 @@ The async parallel methods are:
 * `.parallel()` *Used as an "async with" context manager.*
 * `.get_response()`
 * `.next_response()`
+
+## Supported async libraries
+
+You can use `AsyncClient` with any of the following async libraries.
+
+!!! tip
+    You will typically be using `AsyncClient` in async programs that run on `asyncio`. If that's the case, or if you're not sure what this is all about, you can safely ignore this section.
+
+### [asyncio](https://docs.python.org/3/library/asyncio.html) (Default)
+
+By default, `AsyncClient` uses `asyncio` to perform asynchronous operations and I/O calls. It's a safe default: `asyncio` is built into the standard library and has widespread usage.
```

Comment:
Do we really need the "safe default" part? It's like you're telling people not to use your work on the trio backend :) My humble opinion is that the tip above is enough, and that people already using asyncio will stick to asyncio anyways.

Reply:
Good point, this is a relic from me trying to find a phrasing for « if you don’t care about all this, just use the default », but the introduction paragraph fits this purpose now. I’ll remove this bit.